src/components/ProtectedRoute.jsx

```diff
@@ -16,15 +16,11 @@ const ProtectedRoute = ({ children }) => {
         const sessionUser = data?.session?.user;
```

Comment:
Blocking: Move this sessionUser to `AuthContext`. All of the data related to authenticated user need to be exposed from context.

Reply:
I put it in AuthContext before but it not run, the data return is null. I dont know why.